RELEASE.md

```diff
@@ -0,0 +1,117 @@
+#    contributors may be used to endorse or promote products derived
+#    from this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS ``AS IS'' AND ANY
+# EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+# PURPOSE ARE DISCLAIMED.  IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+# CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+# EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+# PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+# PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+# OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+-->
+
+# Release Notes for 2.26.0
+
+## New Freatures and Improvements
+
+* Added 
+  [developer tools Github repository](https://github.com/triton-inference-server/developer_tools) 
+  that provides a simplified interface for users to interact with the 
+  [Triton Core](https://github.com/triton-inference-server/core) shared library. 
+  These developer tools are in beta and are subject to change.
+
+* Added 
+  [CPU metrics](https://github.com/triton-inference-server/server/blob/r22.09/docs/user_guide/metrics.md#cpu-metrics) 
+  reporting in Triton’s Prometheus metrics endpoint.
+
+* Added 
+  [logging protocol extension](https://github.com/triton-inference-server/server/blob/r22.09/docs/protocol/extension_logging.md) 
+  for users to change logging configuration dynamically.
+
+* Users can specify the custom plugins to be loaded for TensorRT backend through 
+  [command line option](https://github.com/triton-inference-server/tensorrt_backend/blob/r22.09/README.md#command-line-options) 
+  in addition to `LD_PRELOAD`.
+
+* Enabled 
+  [auto-completion for OpenVINO backend](https://github.com/triton-inference-server/openvino_backend/tree/r22.09#auto-complete-model-configuration).
+
+* Enabled Python backend to 
+  [log messages through Triton’s logger](https://github.com/triton-inference-server/python_backend#logging).
+
+* Refer to the 22.09 column of the 
+  [Frameworks Support Matrix](https://docs.nvidia.com/deeplearning/frameworks/support-matrix/index.html) 
+  for container image versions on which the 22.09 inference server container 
+  is based.
+
+* Added 
+  [quick search](https://github.com/triton-inference-server/model_analyzer/blob/main/docs/config_search.md#quick-search-mode) 
+  algorithm to Model Analyzer to drastically reduce search time.
+
+* Added 
+  [GPU metrics](https://github.com/triton-inference-server/server/blob/main/docs/user_guide/perf_analyzer.md#server-side-prometheus-metrics) 
+  gathering to Perf Analyzer, which is also used by Model Analyzer to improve 
+  accuracy of those metrics.
+
+
+## Known Issues
+
```

Comment:
We should document the nullptr issue we investigated. @tanmayv25 Can you provide a relnote for this?

Reply:
" In certain rare cases with specific backends, triton server may crash with segmentation fault while exiting. Preliminary analysis show that there might be a race condition in clean up of backend/model/instance state objects. Exact root cause is still unknown."

Reply:
@mc-nv  @nv-kmcgill53

Reply:
Fine with me

Reply:
@tanmayv25 fixed